cmd/inspr/cli/cli.go

```diff
@@ -33,15 +34,21 @@ func NewInsprCommand(out, err io.Writer, version string) *cobra.Command {
 }
 
 func mainCmdPreRun(cm *cobra.Command, args []string) error {
+	if cm.Name() == "init" {
```

Comment:
@ptcar2009 interesting, if the command is init, we must do nothing since the CLI doesn't have a config.